bats/tests/helpers/vm.bash

```diff
@@ -59,7 +43,7 @@ EOF
     if [ "$RD_USE_IMAGE_ALLOW_LIST" != "false" ]; then
         RD_USE_IMAGE_ALLOW_LIST=true
     fi
-
```

Comment:
Leave the empty line because this code is logically separate from what came before.

Reply:
done